pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -84,9 +84,6 @@ type RepositorySpec struct {
 	// Edit options within the repository
 	Editing EditingOptions `json:"editing"`
 
-	// DeletePolicy options within the repository
-	DeletePolicy DeletePolicy `json:"deletePolicy"`
```

Comment:
better to use standard pattern with https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/